### DISCUSSION TITLE
Primeiro Projeto - iniciante

### DISCUSSION THREAD

**Discussion by djteix** [frontendbr/forum#1516] [Discussão]
Olá a todos, essa é a primeira comunidade que tenho contato como newbie developer, então queria deixar meu agradecimento a todos.
Estou estudando firme HTML5,CCS, JS e NODE JS, e penso após me sentir bem seguro fazer meu primeiro projeto envolvendo essas tecnologias para treinar e ser meu ponta pé inicial e dizer que eu já programei algo. ahahahhah!
Minha duvidas são as seguintes:
1)As tecnologias mencionadas acima são boas para um primeiro projeto ou eu deveria estudar um pouco mais, como um framework?
2)Se as tecnologias estão ok para um primeiro projetinho, alguém poderia me sugerir um bom desafio inicial que envolva todas as tecnologias acima faladas, para que eu possa fazer um prova de conceito das minhas capacidades?

Obrigado a todos!

**Comment by yuritoledo** [comment]
Como está o seu conhecimento em Css isolado
E em javascript isolado? 

Acho que podemos começar por aí. Digo, o que você já manda bem, só com Css e HTML? Posicionamento, cores, aninhamento e prioridade do Css... Como estão esses caras pra você? 

E ao javascript, o que já manja bem? Funções, variáveis, algoritmos, lógica. Está mandando bem nesses itens? 

Nem todo projeto, quando se está estudando, precisa abrangir todas as 3 tecnologias. 

Você pode fazer um blog responsivo só com Css e HTML, por exemplo 
Ou recriar um site que conhece, só pra testar 

Com javascript você consegue rodar os arquivos e testar em componentes (simples de HTML mesmo)
Um formulário de contato por exemplo. Pegue nome e-mail e mensagem e coloque em um objeto e clique em mostre os juntos, em outra div ou no console do navegador 

Não sei se fui claro, qualquer coisa manda aí haha 

Boa sorte Man o/

**Comment by djteix** [comment]
Você me deu umas ideias boas. Acho vou implementar um site só com HTML e CSS, essa será a versão 1.0. Após, implementarei novos recursos com JS. Sei também que JS é implementado no lado do servidor, mas ainda não sei como, pois estou estudando node js ainda. Ficará para o futuro. haha.

**Comment by yuritoledo** [comment]
Bacana!
Aqui tem alguns exemplos de projetos que você pode criar:
https://learn.freecodecamp.org/responsive-web-design/responsive-web-design-projects/
No primeiro item da lista, tem o objetivo, ou seja, como ele tem que ficar. É bacana para se basear :)

Você consegue fazer muita, mas MUITA coisa só com o client-side (HTML, CSS, JS) para fins de portfólio.

Estou buscando alguém para que eu possa mentorar, se quiser, podemos nos ajudar ;)

Olha esse cara que cara que bacana, pra tu ir se guiando:
https://github.com/kamranahmedse/developer-roadmap#frontend-roadmap

**Comment by djteix** [comment]
Rapaz eu me candidato com as duas mãos, no momento estou sendo guiado pela curiosidade e o poder do google. hahahah.
Como faço pra maior contato mentor? :D

**Comment by yuritoledo** [comment]
Opa :D

Me manda um email: yuriwtoledo@gmail.com que lá trocamos figurinhas!

**Comment by felipefialho** [comment]
Fala @djteix, beleza?

Uns anos atrás fui mentor pra umas pessoas e criei um [repositório com algumas coisas que passava pra elas](https://github.com/felipefialho/felipefriends), já é um pouco antigo, mas de repente pode servir de base pra ti.

**Comment by mutheusalmeida** [comment]
Olá, @djteix. Eu recomendo o currículo e projetos da freeCodeCamp. Termine o currículo e os projetos, crie um portfolio, e adicione os projetos no portfolio. Depois, pegue projetos pequenos como freelancer, ótimo para conseguir experiência e uma graninha. Também tente replicar sites com uma abordagem diferente e melhor.

**Comment by henriquegibi** [comment]
@yuritoledo como vai? Gostaria de ser mentorado também. Desculpa o abuso, rsrs! Eu tenho estudado por conta, perdido. Apesar de eu "achar" que estou nos eixos, tenho minhas dúvias.

No momento estou fazendo este curso: [Curso Desenvolvedor Web Completo](https://www.udemy.com/curso-desenvolvedor-web-completo/):

- HTML
- CSS
- JavaScript
- jQuery
- Bootstrap
- EcmaScript
- Youtube API
- SEO
- PHP
- MySQL
- AJAX
- Json
- Webpack

Em seguida, pensava em fazer os seguites cursos, em ordem:

- [Introdução ao Web Services](https://www.udemy.com/course/introducao-web-services/)
- [Curso Completo PHP 7](https://www.udemy.com/curso-php-7-online/)
- [Banco de Dados Completo](https://www.udemy.com/bancos-de-dados-relacionais-basico-avancado/)

Já fiz outros cursos, mas bem aleatoriamente, rsrs. Posso te mandar um email?

**Comment by yuritoledo** [comment]
@henriquegibi bom dia cara!
Sim, fico feliz se puder ajudar. Tenho mais conhecimento com javascript (no front e no back), mas o que eu puder ajudar, conte comigo :)

Me mande um email e vamos bater um papo, quem sabe eu possa te auxiliar :grin:
